tmt/steps/provision/podman.py

```diff
@@ -24,8 +23,14 @@ def __init__(self, data, step):
         self.container_name = None
         self.container_id = None
 
+        # Environment variables compatible with commands used
+        self.podman_env = [f'-e {env}' for env in self.opt('environment')]
+        self.shell_env = ' '.join(self.opt('environment'))
+
     def podman(self, command, **kwargs):
-        return self.run(f'podman {command}', **kwargs)[0].rstrip()
+        """ Run given command via podman """
+        command.insert(0, 'podman')
+        return self.run(command, shell=False, **kwargs)[0].rstrip()
```

Comment:
I'd prefer `return self.run(['podman'] + command, shell=False, **kwargs)[0].rstrip()` so that it won't mutate `command`.

Reply:
agreed, fixed, thx